vispy/visuals/volume.py

```diff
@@ -749,6 +779,17 @@ def threshold(self, value):
             self.shared_program['u_threshold'] = self._threshold
         self.update()
 
+    @property
+    def attenuation(self):
+        """The attenuation rate to apply for the attenuated mip render method."""
+        return self._attenuation
+
+    @attenuation.setter
+    def attenuation(self, value):
+        self._attenuation = float(value)
+        self.shared_program['u_attenuation'] = self._attenuation
```

Comment:
This should probably have the `if 'u_attenuation' ...` like threshold's setter has.